modules/gumgumBidAdapter.js

```diff
@@ -316,6 +316,7 @@ function buildRequests(validBidRequests, bidderRequest) {
     const { currency, floor } = _getFloor(mediaTypes, params.bidfloor, bidRequest);
     const eids = getEids(userId);
     const gpid = deepAccess(ortb2Imp, 'ext.gpid') || deepAccess(ortb2Imp, 'ext.data.pbadslot');
+    const paapiEligible = deepAccess(ortb2Imp, 'ext.ae') === 1
```

Comment:
lines 220 and 295 above have jsdoc warnings

Reply:
@patmmccann , thank you for your review. I have addressed the above warnings. It should be ready to go.